benchmarks/bayesmark/optuna_optimizer.py

```diff
@@ -0,0 +1,117 @@
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Union
+
+import numpy as np
+
+from bayesmark.abstract_optimizer import AbstractOptimizer
+from bayesmark.experiment import experiment_main
+import optuna
+from optuna import pruners
+from optuna import samplers
+
+
+_SAMPLERS = {
+    "GridSampler": samplers.GridSampler,
+    "RandomSampler": samplers.RandomSampler,
+    "TPESampler": samplers.TPESampler,
+    "CmaEsSampler": samplers.CmaEsSampler,
+    "NSGAIISampler": samplers.NSGAIISampler,
```

Comment:
```suggestion
    "NSGAIISampler": samplers.NSGAIISampler,
    "QMCSampler": samplers. QMCSampler,
```

Can we add QMCSampler? Or NO to support older versions of Optuna?

Reply:
Added QMCSampler as well as samplers from integration modules in 440d42a49df6053b92c323bceb62dea12f0481ce.